spec/pom.xml

```diff
@@ -25,7 +25,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>jakarta.enterprise.deploy</groupId>
     <artifactId>deployment-spec</artifactId>
-    <version>1.7-SNAPSHOT</version>
+    <version>1.8-SNAPSHOT</version>
```

Comment:
Shouldn't this be "1.7" for the final release?

Reply:
We are changing versions in release jobs. In the branch we have current version, which is a snapshot after a release. I think that 1.8-SNAPSHOT is the correct value.

Reply:
The final Spec version should be 1.7, the API should be 1.7.2 (since the current version in Maven is 1.7.1).

Reply:
@kwsutter I am actually not sure that the final spec version should be 1.7. In `pom.xml` it says `<spec.version>1.2</spec.version>`. I also couldn't find anything about spec versions higher than 1.2 on the internet. It's [JSR-88](https://jcp.org/en/jsr/detail?id=88) is it?

Reply:
Sigh.  It looks like the version number for this project was set incorrectly back
in the original javaee repository.  We now have an API jar file in Maven with
version 1.7.1.  It would be weird to have a new API jar file with version 1.7.2
and a spec with version 1.2.  But I don't think we can roll back the API jar file
version number.

We may have to bite the bullet, admit our mistake, and set the spec version
number to 1.7.

@kwsutter what do you think?